### PR TITLE
Sylius Recipes Enhancements

### PR DESCRIPTION
**Pull request by Rafikooo**
| Q             | A
| ------------- | ---
| License       | MIT

Continuation of: https://github.com/symfony/recipes-contrib/pull/1723

### CODE REVIEW

**Comment by diimpp** [comment]
@Rafikooo please move `fos_rest` config to a separate file from the previous PR.

image

**Comment by Rafikooo** [comment]
> @Rafikooo please move `fos_rest` config to a separate file from the previous PR.
> 
> image

Hey @diimpp,

Could you explain why this is important?

The reason we proposed combining it into a single file is to prevent potential issues. For example, placing this configuration in a separate file, such as `fos_rest.yaml`, could potentially overwrite someone’s existing configuration. Or perhaps you meant placing it in a file like `sylius_fos_rest.yaml` to make the configuration more explicit? Wdyt?

File: sylius/paypal-plugin/1.7/config/packages/sylius_paypal.yaml [hunk 1]
```diff
@@ -0,0 +1,2 @@
+imports:
+    - { resource: "@SyliusInvoicingPlugin/Resources/config/config.yaml" }
```
Comment (mpysiak) [inline sylius/paypal-plugin/1.7/config/packages/sylius_paypal.yaml:2]:
```suggestion
    - { resource: "@SyliusPayPalPlugin/Resources/config/config.yaml" }
```

**Review by mpysiak — APPROVED** [review]

File: sylius/paypal-plugin/1.7/config/routes/sylius_paypal.yaml [hunk 2]
```diff
@@ -0,0 +1,12 @@
+sylius_paypal_admin:
+    resource: "@SyliusPayPalPlugin/Resources/config/admin_routing.yml"
+    prefix: /admin
```
Comment (diimpp) [inline sylius/paypal-plugin/1.7/config/routes/sylius_paypal.yaml:3]:
```suggestion
    prefix: /%sylius_admin.path_name%
```

**Review by diimpp — APPROVED** [review]

**Comment by diimpp** [comment]
Merge is blocked due `sylius/refund` recipe added for unreleased version -- 2.0

**Review by diimpp — APPROVED** [review]

**Pull request merged**